nagios/tests/test_nagios.py

```diff
@@ -112,6 +113,39 @@ def test_continuous_bulk_parsing(self, aggregator):
         log_file.close()
         assert len(aggregator.events) == ITERATIONS * 503
 
+    def test_create_event_tags(self):
+        """
+        Tags should have proper format otherwise 'Nagios Service Check.groovy' won't get health state correctly
+        """
+        config, nagios_cfg = get_config('\n'.join(["log_file={0}".format(NAGIOS_TEST_LOG)]), events=True)
+        nagios = NagiosCheck(CHECK_NAME, {}, {}, instances=config['instances'])
+        nagios.get_topology = mocked_topology
+        nagios.check(config['instances'][0])
+        nagios_tailer = nagios.nagios_tails[nagios_cfg.name][0]
+
+        event_type = 'SERVICE NOTIFICATION'
+        fields = EVENT_FIELDS.get(event_type, None)
+        parts = [
+            'nagiosadmin',
+            'nagios4',
+            'Root Partition',
+            'CRITICAL',
+            'notify-service-by-email',
+            'DISK CRITICAL - free space: / 1499 MB (2.46% inode=77%):'
+        ]
+        event = nagios_tailer.create_event(
+            timestamp=1603813628, event_type=event_type, hostname='docker-desktop', fields=fields._make(parts)
+        )
+
+        assert event['tags'] == [
```

Comment:
Can we assert the other fields of the event as well

Reply:
Done